synfig-studio/src/gui/docks/dock_timetrack2.cpp

```diff
@@ -233,6 +233,74 @@ void Dock_Timetrack2::setup_tool_palette()
 		action_button_map[tool_button->get_name()] = tool_button;
 		tool_item_group->add(*tool_button);
 	}
+    Gtk::RadioButtonGroup interpolation_group;
+    //button1
+	Gtk::ToolButton *clamped_button = manage(new Gtk::ToolButton(
+													*manage(new Gtk::Image(
+																Gtk::StockID("synfig-interpolation_type_clamped"),
+																Gtk::IconSize::from_name("synfig-small_icon_16x16") )),
+																"_Clamped" ));
+	clamped_button->signal_clicked().connect( [this](){
+
+		current_widget_timetrack->set_interpolation(synfig::INTERPOLATION_CLAMPED);
+
+	} );
+	tool_item_group->add(*clamped_button);
+
+	//button2
+	Gtk::ToolButton *tcb_button = manage(new Gtk::ToolButton(
+													*manage(new Gtk::Image(
+																Gtk::StockID("synfig-interpolation_type_tcb"),
+																Gtk::IconSize::from_name("synfig-small_icon_16x16") )),
+																"_TCB" ));
+
+	tcb_button->signal_clicked().connect( [this](){
+
+		current_widget_timetrack->set_interpolation(synfig::INTERPOLATION_TCB);
+
+	} );
+
+	tool_item_group->add(*tcb_button);
+    //button3
+	Gtk::ToolButton *const_button = manage(new Gtk::ToolButton(
+													*manage(new Gtk::Image(
+																Gtk::StockID("synfig-interpolation_type_const"),
+																Gtk::IconSize::from_name("synfig-small_icon_16x16") )),
+																"_Constant" ));

```

Comment:
did you mean `_("Constant")`? This will translate "`Constant`".
Look other similar cases here.

Reply:
oh yes the gettext, I didn't know about it back then. thanks for the notifying :).